stdlib/unittest/mock.pyi

```diff
@@ -162,7 +159,7 @@ class NonCallableMock(Base, Any):
     side_effect: Any
     called: bool
     call_count: int
-    call_args: Any
+    call_args: _Call | None
```

Comment:
```suggestion
    call_args: _Call | MaybeNone
```
where `from _typeshed import MaybeNone`

I think the primer suggests this is worth doing